### PR TITLE
Replace intro training slides with videos

### LINKED ISSUE #2574: Training: Replace slides with videos
For the 2 intro modules, `Why CockroachDB?` and `CockroachDB Architecture`, replace the slides with videos of tech talks.

Also cross-reference our architecture docs.

### PR DESCRIPTION
**Pull request by jseldess**
Fixes #2574

### CODE REVIEW

File: v1.1/training/architecture-overview.md [hunk 1]
```diff
@@ -6,7 +6,12 @@ sidebar_data: sidebar-data-training.json
 redirect_from: /training/architecture-overview.html
 ---
 
-<iframe src="https://docs.google.com/presentation/d/e/2PACX-1vRtr1hx7IJuM7w1ODo9FfR7boTbBHkmIzYHEiJ-HQi9YQVeNhb7WsBdP0wBSCpt62XnkzxVrnYPiFkO/embed?start=false&loop=false" frameborder="0" width="756" height="454" allowfullscreen="true" mozallowfullscreen="true" webkitallowfullscreen="true"></iframe>
+Watch the rest Alex Robinson's talk, where he explains the CockroachDB architecture and how it was built.
```
Comment (a-robinson) [inline v1.1/training/architecture-overview.md]: "the rest of ..."

Comment (jseldess) [reply v1.1/training/architecture-overview.md]: Done.

**Comment by jseldess** [comment]
TFTR, @a-robinson. Yep, I think that's what we'll do for in-person trainings.

**Comment by a-robinson** [comment]
Yeah, the only real alternative on my mind would be to let people select between Slides vs Video the same way we have selectors for Secure vs Insecure elsewhere in the docs.

**Comment by jseldess** [comment]
I put a video/slides toggle on these pages. Thanks for that idea, @a-robinson!

File: v1.1/training/architecture-overview.md [hunk 2]
```diff
@@ -6,8 +6,26 @@ sidebar_data: sidebar-data-training.json
 redirect_from: /training/architecture-overview.html
 ---
 
+Watch the rest of Alex Robinson's talk, where he explains the CockroachDB architecture and how it was built.
```
Comment (a-robinson) [inline v1.1/training/architecture-overview.md]: Nit: I'd put this sentence inside the Video div.

File: v1.1/training/why-cockroachdb.md [hunk 3]
```diff
@@ -5,8 +5,27 @@ sidebar_data: sidebar-data-training.json
 redirect_from: /training/why-cockroachdb.html
 ---
 
+Kick off your training by watching Alex Robinson, a CockroachDB Engineer, explain the history of databases and why CockroachDB was built.
```
Comment (a-robinson) [inline v1.1/training/why-cockroachdb.md]: Nit: I'd put this sentence inside the Video div.

File: v1.1/training/architecture-overview.md (hunk 2 above)
Comment (jseldess) [reply v1.1/training/architecture-overview.md]: I've left it at the top but mentioned the slides as well.

File: v1.1/training/why-cockroachdb.md (hunk 3 above)
Comment (jseldess) [reply v1.1/training/why-cockroachdb.md]: Same as above.

**Pull request merged**